src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -1631,8 +1612,9 @@ bool ColorControlServer::moveToHueCommand(app::CommandHandler * commandObj, cons
  */
 bool ColorControlServer::moveToHueAndSaturationCommand(app::CommandHandler * commandObj,
                                                        const app::ConcreteCommandPath & commandPath, uint16_t hue,
-                                                       uint8_t saturation, uint16_t transitionTime, uint8_t optionsMask,
-                                                       uint8_t optionsOverride, bool isEnhanced)
+                                                       uint8_t saturation, uint16_t transitionTime,
+                                                       chip::BitMask<OptionsBitmap> optionsMask,
+                                                       chip::BitMask<OptionsBitmap> optionsOverride, bool isEnhanced)
```

Comment:
```suggestion
                                                       BitMask<OptionsBitmap> optionsMask,
                                                       BitMask<OptionsBitmap> optionsOverride, bool isEnhanced)
```
I thought you said you fixed all of these....